app/views/conference_users/_table.html.haml

```diff
@@ -1,9 +1,9 @@
 %table.zebra-striped
   %thead
     %th.first
-    %th= t('activerecord.attributes.user.email')
-    %th= t('activerecord.attributes.conference.acronym')
-    %th= t('activerecord.attributes.conference_user.role')
+    %th= User.human_attribute_name('email')
+    %th= Conference.human_attribute_name('acronym')
+    %th= Conference_user.human_attribute_name('role')
```

Comment:
`Conference_user` is not a valid class name in respect to Ruby conventions.  Typo?

Reply:
Commit fixed. Thanks1